src/processTargets.ts

```diff
@@ -18,6 +18,7 @@ import {
 import { performInsideOutsideAdjustment } from "./performInsideOutsideAdjustment";
```

Comment:
Looks like there are some changes to behaviour here so would be great to add a test or two to capture what's changed

Reply:
Tests added